devtools/packages.yaml

```diff
@@ -0,0 +1,59 @@
+---
+# Javascript packages
+web-server:
+  objective: proxy api, provides html & assets (js)
+  release-command: src/frontend/web_application/bin/release
+  dependencies:
+    nodejs: "^6.10"
+  packages:
+  - "/dist/web-server"
+electron:
+  objective: an app packaged for linux, mac & windows
+  release-command: src/frontend/web_application/bin/release
+  dependencies:
+    nodejs: "^6.10"
+  packages:
+  - "/dist/linux-app"
+  - "/dist/mac-app"
+  - "/dist/windows-app"
+cordova:
+  objective: an app packaged for android, ios & windows phone
+  release-command: src/frontend/web_application/bin/release
+  dependencies:
+    nodejs: "^6.10"
+  packages:
+  - "/dist/android-app"
+  - "/dist/ios-app"
+  - "/dist/windows-phone-app"
+
+# Python packages
+caliopen_storage:
+  objective: Caliopen python storage package
+  release-command: devtools/make_package.sh -t python -s src/backend/main/py.storage -n caliopen_storage
+  dependencies:
+    python: "^2.7"
+  packages:
+  - "/dist/python"
```

Comment:
Difficult to add the version number in the filename and not well suited a directory per python package.

What about a /dist/<release_number>/<package_name>